sonaranalyzer-dotnet/rspec/cs/S3353_c#.json

```diff
@@ -0,0 +1,15 @@
+{
+  "title": "Unchanged local variables should be \"const\"",
+  "type": "CODE_SMELL",
+  "status": "ready",
+  "remediation": {
+    "func": "Constant\/Issue",
+    "constantCost": "2min"
+  },
+  "tags": [
+    
+  ],
+  "defaultSeverity": "Critical",
```

Comment:
It might be worth reconsidering the default severity: `Critical` for this looks overkill.